main.css

```diff
@@ -1,36 +1,37 @@
+@charset "utf-8";
 :root {
-  --body-height: 556px;
```

Comment:
@Viet281101 I think you forgot to undo the changes here in the CSS stylesheet but no worries, we can forget this. 👍